website/src/controllers/api-v3/groups.js

```diff
@@ -18,6 +18,7 @@ import * as firebase from '../../libs/api-v3/firebase';
 import { sendTxn as sendTxnEmail } from '../../libs/api-v3/email';
 import { encrypt } from '../../libs/api-v3/encryption';
 import { quests as questScrolls } from '../../../../common/script/content';
+import { mockAnalyticsService } from '../../libs/api-v3/analyticsService';
```

Comment:
That comments said to use this in a non production environment. I was thinking that that this was replaced somehow in production. Maybe I just misread that.